hack/local-up-cluster.sh

```diff
@@ -532,9 +533,16 @@ function start_apiserver {
     kube::util::create_client_certkey "${CONTROLPLANE_SUDO}" "${CERT_DIR}" 'client-ca' kube-aggregator system:kube-aggregator system:masters
     kube::util::write_client_kubeconfig "${CONTROLPLANE_SUDO}" "${CERT_DIR}" "${ROOT_CA_FILE}" "${API_HOST}" "${API_SECURE_PORT}" kube-aggregator
 
+    cloud_config=""
```

Comment:
While it won't directly cause an issue, this creates both an uppercase `CLOUD_CONFIG`, and a lowercase `cloud_config` with very different uses. Makes it less readable/clear. Perhaps "cloud_provider_arg" or something slightly different?

Reply:
Thanks fixed!